lldb/source/Target/TargetProperties.td

```diff
@@ -102,7 +102,7 @@ let Definition = "target" in {
     DefaultUnsignedValue<1024>,
     Desc<"Maximum number of characters to show when using %s in summary strings.">;
   def MaxMemReadSize: Property<"max-memory-read-size", "UInt64">,
-    DefaultUnsignedValue<1024>,
+    DefaultUnsignedValue<4294967295>,
```

Comment:
suggestion: Either make it hex, or instead of providing a default value here, initialize the CommandOption to UINT32_MAX